Tasks/KubernetesV0/src/clusterconnection.ts

```diff
@@ -71,6 +80,20 @@ export default class ClusterConnection {
         fs.writeFileSync(this.kubeconfigFile, kubeconfig);
     }
 
+    private createKubeconfig(kubernetesEndpoint: string): void {
```

Comment:
Move this to common file. It used in V0 and V1 task.

Reply:
done